### PR TITLE
[Add] Support for CommonFileStore

### PR DESCRIPTION
**Pull request by lxatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
[Add] Support for CommonFileStore

### CODE REVIEW

File: CDP4Composition/Mvvm/MenuItems/RibbonButtonEngineeringModelDependentViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,226 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="RibbonButtonEngineeringModelDependentViewModel.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
```
Comment (samatrhea) [inline CDP4Composition/Mvvm/MenuItems/RibbonButtonEngineeringModelDependentViewModel.cs:3]: 2023

File: CDP4Composition/Mvvm/MenuItems/SessionEngineeringModelMenuGroupViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿// -------------------------------------------------------------------------------------------------
+// <copyright file="SessionEngineeringModelSetupMenuGroupViewModel.cs" company="RHEA System S.A.">
+//   Copyright (c) 2017 RHEA System S.A.
```
Comment (samatrhea) [inline CDP4Composition/Mvvm/MenuItems/SessionEngineeringModelMenuGroupViewModel.cs:3]: please use correct header

File: EngineeringModel/ViewModels/Dialogs/CommonFileStoreDialogViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="CommonFileStoreDialogViewModel.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
```
Comment (samatrhea) [inline EngineeringModel/ViewModels/Dialogs/CommonFileStoreDialogViewModel.cs:3]: 2023

**Review by samatrhea — CHANGES_REQUESTED** [review]
minor changes requested on file headers, code looks good, no need to pass by again

**Review by jaimeatrhea — APPROVED** [review]

**Pull request merged**